Common/Kernel/CommonValidations.cs

```diff
@@ -13,5 +13,18 @@ public static void ValidateId(int id)
                 throw new Exception(Errors.IdCanNotBeLessThanZero);
             }
         }
+        
+        public static void ValidateEmail(string email)
+        {
+
+            var addr = new MailAddress(email); 
+            bool emailIsValid = addr.Address == email;

```

Comment:
Wow wow wow, do we really need this variable???

Reply:
fixed